src/main/java/com/codegym/jrugotom5/controller/AdvertController.java

```diff
@@ -0,0 +1,30 @@
+package com.codegym.jrugotom5.controller;
+
+import com.codegym.jrugotom5.entity.Advert;
+import com.codegym.jrugotom5.service.AdvertService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Date;
+import java.util.List;
+
+@RestController
+@RequestMapping("/adverts")
+public class AdvertController {
+    AdvertService advertService;
+    public AdvertController( @Autowired AdvertService advertService ) {
+        this.advertService = advertService;
+    }
+
```

Comment:
i think better would be to use RequiredArgsConstructor annotation, not Autowired.

Reply:
Done